test/e2e/storage/testsuites/provisioning.go

```diff
@@ -690,3 +710,49 @@ func prepareDataSourceForProvisioning(
 
 	return dataSourceRef, cleanupFunc
 }
+
+func preparePVCDataSourceForProvisioning(
+	node framework.NodeSelection,
+	client clientset.Interface,
+	dynamicClient dynamic.Interface,
+	source *v1.PersistentVolumeClaim,
+	class *storagev1.StorageClass,
+) (*v1.TypedLocalObjectReference, func()) {
+	var err error
+	if class != nil {
+		ginkgo.By("[Initialize dataSource]creating a StorageClass " + class.Name)
+		_, err = client.StorageV1().StorageClasses().Create(class)
+		framework.ExpectNoError(err)
+	}
+
+	ginkgo.By("[Initialize dataSource]creating a source PVC")
+	sourcePVC, err := client.CoreV1().PersistentVolumeClaims(source.Namespace).Create(source)
+	framework.ExpectNoError(err)
+	err = framework.WaitForPersistentVolumeClaimPhase(v1.ClaimBound, client, sourcePVC.Namespace, sourcePVC.Name, framework.Poll, framework.ClaimProvisionTimeout)
```

Comment:
To support delayed binding, we should create the pod first, and then we can wait for pvc bound.

Reply:
Maybe better to check for delayed binding and react appropriately; then the calling test can sort out what to do (it may check the same way).  My thought being that rather than just making the helper in the suite work in all conditions, make sure it's expclicitly checking based on the bind mode?

Reply:
Unsure if this test really cares about the different between delayed and immediate.  For most tests I think it'll be simpler to handle both the same way